modules/react-components/src/components/alert/wizard-alert.tsx

```diff
@@ -47,7 +51,9 @@ type UseWizardAlertHookReturnType = [ WizardAlert, React.Dispatch<React.SetState
  *
  * @returns An array containing the state, setState function and the alert React element.
  */
-export const useWizardAlert = (): UseWizardAlertHookReturnType => {
+export const useWizardAlert = (props?: WizardAlertProps): UseWizardAlertHookReturnType => {
+
+    const { ["data-componentid"]: componentId } = props || { "data-componentid": "wizard-alert" };
```

Comment:
Lets define the default props in the method signature itself.

Eg:

```js
export const useWizardAlert = (props: WizardAlertProps = { "data-componentid": "wizard-alert" }): 
    UseWizardAlertHookReturnType => {

    const { ["data-componentid"]: componentId } = props;
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/6597.